### ISSUE TITLE
Wouldn't this be more convenient in a single module?

### ISSUE THREAD

**Issue by gaearon** [Reactive-Extensions/RxJS-Modular#4]
I mean, I get that it's the point of this package.. But look at how `lodash` does it.

There's a single package, but you are encouraged to `require` individual modules which are CommonJS so you only get the shared common denominator of what you `require`d.

I think Rx should work similarly, shouldn't it?

**Comment by gaearon** [comment]
This is becoming a concern for me.

I really want to depend on disposables from Rx in my lib, but I can't drag even rx-lite just for them.
OTOH I don't like each-module-as-a-package for the reasons described above. It's a suboptimal solution.

Finally, these packages are in too alpha stage compared to the real Rx code (some of them are not registered on NPM at all despite them having `package.json`, some have serious typos and mistakes).

If this experiment is not maintained, I'm going to make a package with just Rx disposables. But it sucks to duplicate the effort.

**Comment by gaearon** [comment]
I published [`disposables`](https://github.com/gaearon/disposables) on NPM in case anybody else needs them.

**Comment by andreypopp** [comment]
I fully agree with @gaearon's point, the one package composed of small modules is best solution. It provides modularity without huge overhead of maintaining and using multiple packages and their versions.
